www/js/models/platforms.js

```diff
@@ -6,31 +6,58 @@
 
 /*
  * Copyright (c) 2015, Joyent, Inc.
+ * Copyright 2023 MNX Cloud, Inc.
  */
 
 var Backbone = require('backbone');
-var _ = require('underscore');
+
+// Sort by os, then platform image version
+function sortPlatforms (a, b) {
+    if (a.os === b.os) {
+        return a.version > b.version ? -1 : 1;
+    }
+    return a.os > b.os ? -1 : 1;
```

Comment:
Does this sort order ensure that SmartOS platforms are listed first for unsetup CNs? (that seems to be the case...)

Reply:
Yes, I made sure that was the case. That will be the behavior on both unsetup CNs and on the _Default Server Boot Options_ dropdown.